sql/core/src/test/scala/org/apache/spark/sql/test/SharedSparkSession.scala

```diff
@@ -111,7 +111,7 @@ trait SharedSparkSession
     spark.sharedState.cacheManager.clearCache()
     // files can be closed from other threads, so wait a bit
     // normally this doesn't take more than 1s
-    eventually(timeout(10.seconds)) {
+    eventually(timeout(10.seconds), interval(2.seconds)) {
```

Comment:
what's the problem if interval is small?

Reply:
`assertNoOpenStreams` and `removeOpenStream` are both synchronized on `openStreams` map. I want to increase interval to prevent starvation for `removeOpenStream`.

Reply:
The default is 15 milliseconds. It should be fine. 

But, I am also fine to change this to 2 seconds. I checked the test history and this does not increase the testing time.